source/docs/essentials/installation.md

```diff
@@ -117,9 +126,16 @@ See [our caching section](/docs/advanced/graphql/dataloaders-and-cache-invalidat
 
 Launch your Front-Commerce project in development mode by running:
 
-```sh
-npm run start
-```
+- For node < 17
+
+  ```sh
+  npm run start
+  ```
+
+- For node >= 17
```

Comment:
I still don't get why we put so much content in our documentation for a version that will be unsupported in the end of the month and probably not used by anybody.

Reply:
this will be the same with node 18 which is covered by the >=17 remark